### DISCUSSION TITLE
Possibillity to adjust gap/distance between taskbar and start menu either with slider or percentage

### DISCUSSION THREAD

**Discussion by Jess94QMRNS** [Open-Shell/Open-Shell-Menu#834] [Ideas]
Noticed this here problem today after customizing the taskbar,shell and start menu

Used openshell for the start menu, used uxthemepatcher for the shell and used retrobar for the taskbar

Seems to be a slight gap when using retro bar and openshell between the start menu and taskbar.

Would be nice if there was the possibility to choose where the bottom of the startmenu will be when open either with a slider or with percentages.

![Capture](https://user-images.githubusercontent.com/89818456/131423223-4c65a273-9cfe-48f9-afb8-5a704ae65661.PNG)

**Comment by bonzibudd** [comment]
See https://github.com/Open-Shell/Open-Shell-Menu/issues/558

**Reply by Jess94QMRNS** [reply]
Open shell won't let me enable it, its washed out

![image](https://user-images.githubusercontent.com/89818456/131498916-99cee16e-9770-430a-9a0d-c88453483214.png)

**Reply by bonzibudd** [reply]
That option is for the scenario when the Taskbar is on the side of the screen, you need to update to version 4.4.169 to use the offset or align to working area options.
